README.md

```diff
@@ -32,3 +52,29 @@ var v3 = new Vector(3, 4)
 console.log(v3.getLength());
 // => 5
 ```
+
+
+
+class Vector {
+  constructor (x, y){
+    this.x = x;
+    this.y = y;
+  }
+  plus (el) {
+    return new Vector (this.x + el.x , this.y + el.y)
+  }
+  minus (el) {
+    return new Vector (this.x - el.x , this.y - el.y)
+  }
+  getLength () {
+    return Math.sqrt(Math.pow(this.x, 2) + Math.pow(this.y, 2))
+  }
+}
+
```

Comment:
Solid job. Argument names should be a little clearer as to what they actually are/expected to be. Plus and minus takes in a vector as an argument so vector, anotherVec, vec, etc would be more descriptive.